task2/User.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+include "../vendor/autoload.php";
+
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Mapping\ClassMetadata;
+
+class User
+{
+    private int $id;
+    private string $name;
+    private string $email;
+    private string $password;
+    private DateTime $creationDateTime;
+
+    public function __construct($id, $name, $email, $password)
```

Comment:
Не указан тип у входных параметров 
```suggestion
    public function __construct(int $id, string $name, string $email, string $password)
```